include/ocpp/v201/charge_point.hpp

```diff
@@ -166,6 +161,21 @@ struct Callbacks {
     /// \brief Callback function that can be used to handle arbitrary data transfers for all vendorId and
     /// messageId
     std::optional<std::function<DataTransferResponse(const DataTransferRequest& request)>> data_transfer_callback;
+
+    /* Callbacks for networking */
+    /// \brief register a \p callback that is called when the websocket is connected successfully
+    std::function < void(std::optional<NetworkConnectionProfile> network_profile)> websocket_connected_callback;
+
+    /// \brief register a \p callback that is called when the websocket connection is disconnected
+    std::function<void()>websocket_disconnected_callback;
```

Comment:
This can be optional as well. It is not mandatory to know for the implementing class if the websocket is connected or disconnected.